### PR TITLE
Clean up whitespace in CSS

### PR DESCRIPTION
**Pull request by dcwalk**
Hey, 

Just did a pass to clean up the whitespace in CSS, noticed a few errors that we can probably also resolve, thought I could ping here by line @shaqsingh ...?

### CODE REVIEW

File: assets/main.css [hunk 1]
```diff
@@ -14,7 +14,7 @@ dl, dd, ol, ul, figure {
  * Basic styling
  */
 body {
-	font: 400 16px/1.5 'Source Sans Pro', sans-serif;}
+	font: 400 16px/1.5 'Source Sans Pro', sans-serif;
```
Comment (dcwalk) [inline assets/main.css]: @shaqsingh -- want to make sure to point out this issue, are the colour and background-colour properties still going to be intended?

File: assets/post.css [hunk 2]
```diff
@@ -31,146 +34,165 @@ h1, h2, h3, h4, h5, h6,
...
-
-  #intro {
-      float: left;
-      background-color: #393939;
-      color: white;}
-  #soon {
-      background-color:rgba(79, 187, 169, 0.7);
-      float: left;}
+    background-image:url(assets/CH1.png);
+    }
```
Comment (dcwalk) [inline assets/post.css]: @shaqsingh right now this isn't doing anything... wanted to flag to see if this needs to be corrected or safe to remove

File: assets/post.css [hunk 3]
```diff
@@ -19,10 +22,10 @@ body {
   -moz-font-feature-settings: "kern" 1;
```
Comment (dcwalk) [inline assets/post.css:21]: I'm getting prompted to add `-ms-text-size-adjust:` alongside `-webkit...` when linting, not super familiar with this property...

**Comment by dcwalk** [comment]
I thought the format I chose (3 spaces in front of sections; dangling curly brace, 1 tab and intended properties) was the most popular in the two files but turns out it wasn't

If anyone has a strong preference we can format another way! Just wanted to improve consistency :)

**Comment by dcwalk** [comment]
This potentially resolves #11 if we remove that section from css

File: assets/post.css (hunk 2 above)
Comment (shaqsingh) [reply assets/post.css]: I think I caught this in the figures-spacing branch i just merged

**Review by shaqsingh — APPROVED** [review]

File: assets/post.css (hunk 3 above)
Comment (shaqsingh) [reply assets/post.css:21]: this bit was in the original jekyll post.css file I didn't touch it but that `-ms...` looks like it adds IE support

**Review by shaqsingh — APPROVED** [review]

**Review by shaqsingh — APPROVED** [review]

**Pull request merged**